feature/experimental/p4rt/ate_tests/traceroute_packetin_test/packetin_test.go

```diff
@@ -139,7 +139,7 @@ func testPacketIn(ctx context.Context, t *testing.T, args *testArgs, IsIpv4 bool
 		defer programmTableEntry(leader, args.packetIO, true, IsIpv4)
 	} else {
 		// Insert p4rtutils acl entry on the DUT
-		if err := programmTableEntry(leader, args.packetIO, true, false); err != nil {
+		if err := programmTableEntry(leader, args.packetIO, false, false); err != nil {
```

Comment:
Can you please rebase this PR since I found the base is out of sync.

Reply:
I don't see any difference in the two blocks, why do you need "if IsIpv4 {" and "} else {" ? Do I misunderstand it?